app/Services/ImportExport/SubfleetImporter.php

```diff
@@ -89,4 +95,25 @@ protected function processFares(Subfleet &$subfleet, $col): void
             $fare->save();
         }
     }
+
+    /**
+     * Parse all of the rakns in the multi-format
+     *
+     * @param Subfleet $subfleet
+     * @param          $col
+     */
+    protected function processRanks(Subfleet &$subfleet, $col): void
+    {
+        $ranks = $this->parseMultiColumnValues($col);
+        foreach ($ranks as $rank_id => $rank_attributes) {
+            if (!\is_array($rank_attributes)) {
+                $rank_id = $rank_attributes;
+                $rank_attributes = [];
+            }
+
+            $rank = Rank::firstOrCreate(['id' => $rank_id], ['name' => $rank_id]);
```

Comment:
Can you change thte name to "imported rank {id}"?

Reply:
done !